changelog.d/16156.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in 1.87 where synapse would send an excessive amount of federation requests to servers which have been offline for a long time. Contributed by Nico.
```

Comment:
This says 1.87 but the description says 1.90?

Reply:
I forgot to update the description, I somehow thought https://github.com/matrix-org/synapse/pull/15783 got merged into 1.90 when I wrote the commit message and then later fixed it.